james-jamestall/homework2/99-bottles-of-beer/app.js

```diff
@@ -36,13 +36,13 @@ while (i > -1) {
 	if (i > 1) {
```

Comment:
A `while` loop is a much more concise solution to consume all this beer. Good call. 👍 
It'd be easier to read this code without so much whitespace, but with 99 bottles to get through it's probably fine for now.